models/ideas.js

```diff
@@ -11,18 +11,26 @@ function createIdea(header, description, askedSum, userId) {
     newIdea.description = description;
     newIdea.askedSum = askedSum;
     newIdea.userId = userId;
-    newIdea.approve = 'pending';
+    newIdea.status = 'pending';
 
     const newIdeas = [...ideas, newIdea];
     setIdeas(newIdeas);
     return newIdea.id;
 }
 
 function updateIdea(ideaId, key, value) {
```

Comment:
What if `key` is undefined?

Reply:
throw an Error object with message!
Updated and tested